edgelet/version.txt

```diff
@@ -1 +1 @@
-1.1.13
\ No newline at end of file
+1.1.14
```

Comment:
We don't want to bump the edgelet version, since this is a Docker image-only release.